examples/tests/python/example_test/pkg_resources_access/test_bigquery_python_run.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from google.cloud import bigquery
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+# TODO: ???/explain how you as a user can write pants integration tests, e.g. for binaries you want
```

Comment:
I think that rather than placing it here (which _should_ cause it to get picked up by https://github.com/pantsbuild/pants/blob/master/tests/python/pants_test/projects/test_testprojects_integration.py), you should incorporate it into pants' integration tests somewhere under https://github.com/pantsbuild/pants/tree/master/tests/python/pants_test/backend/python.

Examples should be end-user readable, and I think that in this case this test living here distracts from the fact that it is providing coverage for something.

Reply:
Thank you! I've moved the test into `tests/python/pants_test/backend/python/tasks/test_bigquery_python_run_integration.py`!